src/main/java/com/godaddy/sonar/ruby/RubyPlugin.java

```diff
@@ -1,7 +1,7 @@
 package com.godaddy.sonar.ruby;
 
 import com.godaddy.sonar.ruby.core.Ruby;
-import com.godaddy.sonar.ruby.core.RubySourceCodeColorizer;
+// import com.godaddy.sonar.ruby.core.RubySourceCodeColorizer;
```

Comment:
If you removed the RubySourceCodeColorizer file, the comment reference should be removed as well

Reply:
Yup it should be.....and it's removed.